tests/console/console.go

```diff
@@ -153,6 +153,31 @@ func SecureBootExpecter(vmi *v1.VirtualMachineInstance) error {
 	return err
 }
 
+// BlockDeviceResizeExpecter should be called on a VMI that has block mode
+// It will parse the kernel output (dmesg) and succeed if it finds that the device resize succeeded
+func BlockDeviceResizeExpecter(vmi *v1.VirtualMachineInstance) error {
+	virtClient, err := kubecli.GetKubevirtClient()
+	if err != nil {
+		return err
+	}
+	expecter, _, err := NewExpecter(virtClient, vmi, 10*time.Second)
+	if err != nil {
+		return err
+	}
+	defer expecter.Close()
+
+	b := append([]expect.Batcher{
+		&expect.BExp{R: "resized successfully"},
```

Comment:
can there we a race between the expecter and the VM ?
and we will lose the message if it already stdout in the past, before the test process managed to run the expecter ?

Reply:
from what I understand from a similar expecter it will scan the dmesg for this msg also it it already printed, but I might be mistaken. I ran this several times to see that it catches the print.
I added this expecter because this msg in the CI was received at the test when we start testing for other stuff running bash cmds and expecting empty prompt in response which instead we received this msg which failed the test, the solution was to wait for this msg before continuing to the test. (@rmohr also response to your question)

Reply:
IIUC the expecter checks the stdout, not the dmesg buffer itself
it might also look on the past stdout, and if so, we will be fine

in case you want you can simply add delay of lets say 1 minute and see if you then trigger the expecter it catches 
and historical "resized"